### PR TITLE
chore: add remaining mirrors

### PR DESCRIPTION
**Pull request by lucasfernog**

### CODE REVIEW

**Comment by lucasfernog** [comment]
I'll create the missing repos tomorrow.

**Review by wusyong — APPROVED** [review]

**Comment by FabianLars** [comment]
I went ahead and created the repos. Still had the muscle memory from the other 20 😂 

Do we really need one for single-instance? I made that old repo read-only a while back instead of adding as a mirror because it only had a rust side. Do we want it for consistency too?

**Comment by FabianLars** [comment]
Okay nevermind, i lost access to the single-instance mirror so you gotta do it if you think we should use it again :)

**Comment by lucasfernog** [comment]
We can't really use the mirror for Rust-only deps since we do not resolve the workspace fields in Cargo.toml when mirroring. So we should also deprecate localhost and window-state.

**Comment by lucasfernog** [comment]
Oh window-state has JS API too. Weird :joy:

**Comment by FabianLars** [comment]
window-state recently got js apis, but persisted-scope doesn't and won't

**Comment by FabianLars** [comment]
> Oh window-state has JS API too. Weird 😂

Yeah for undecorated windows where the CloseRequested api isn't triggered

**Comment by lucasfernog** [comment]
Thanks for creating the repos btw. I went on checking which ones were missing and they were all there already, so I was like `wtf???? is there a workflow creating this too?` ofc the workflow is fabian.yml :joy:

File: plugins/mirrors.txt [hunk 1]
```diff
@@ -6,14 +7,17 @@ dialog
 fs
 global-shortcut
 http
-localhost
 log
+os
 persisted-scope
```
Comment (FabianLars) [inline plugins/mirrors.txt]:
```suggestion
```

**Pull request merged**